examples/Volcanotectonic_Iceland/dike_intrusion_locate.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+"""
+This script runs the locate stage for the Iceland dike intrusion example.
+
+"""
+
+import os
+os.environ['OMP_NUM_THREADS'] = '1'
```

Comment:
Why this here, but not in the other scripts?

Reply:
Yeh not sure whether it's best to have this or not. It just sets it such that numpy doesn't hog all available threads - in locate this happens much more than detect / trigger because there are a lot of 'single-threaded' things which take a while - in removing the instrument response, plotting things etc. It's annoying because it looks like QM is ignoring the 'threads' you've given it, when in actual fact it isn't.

Reply:
In that case, we could nip it in the bud in the top level `__init__`? Don't have a gauge for how much impact this will have on the overall execution speed, though?